README.md

```diff
@@ -530,7 +530,7 @@ Services to securely store your Docker images.
 - [Docker Registry v2][distribution] - The Docker toolset to pack, ship, store, and deliver content
 - [Docket](https://github.com/netvarun/docket) - Custom docker registry that allows for lightning fast deploys through bittorrent by [@netvarun](https://github.com/netvarun/)
 - [GCE Container Registry :heavy_dollar_sign:](https://cloud.google.com/container-registry/) Fast, private Docker image storage on Google Cloud Platform
-- [GitLab Container Registry](https://docs.gitlab.com/ce/user/project/container_registry.html) - Repositories focused on using it images in GitLab CI
+- [GitLab Container Registry](https://docs.gitlab.com/ee/user/packages/container_registry/) - Registry focused on using its images in GitLab CI
```

Comment:
GitLab registry is part of CE too: https://docs.gitlab.com/ce/user/packages/container_registry/
we can keep  the link to the community edition instead of enterprise. But tbh I don't know if there is any difference.

Reply:
@dmitrytokarev thank you for pointing this out. I would have reverted / edited the adjustment but gitlab doesn't allow additional changes to an already merged PR, so https://github.com/veggiemonk/awesome-docker/pull/926 has been created.